parm/config/gefs/config.extractvars

```diff
@@ -0,0 +1,34 @@
+#! /usr/bin/env bash                                                                                                                                                                                                                     
+
+########## config.extractvars ##########
+# Extractvars specific
+
+echo "BEGIN: config.extractvars"
+
+. "${EXPDIR}/config.resources" extractvars
+
+export COMPRSCMD=${COMPRSCMD:-bzip2}
+
+export compress_ocn=0 #1: compress extracted ocean product, 0: do not compress extracted ocean product
+export compress_ice=0 #1: compress extracted ice product, 0: do not compress extracted ice product
+
+export ocn_dataformat="netcdf"
+export ice_dataformat="netcdf"
+
+export ocnres="5p00" #Resolution of ocean products
+export iceres="5p00" #Resolution of ice products
+export wavres="5p00" #Resolution of wave products
+
+export FHOUT_WAV_NOSCRUB=6 #Frequency of wave output to be saved on disk
+
+#Paramater Tables used
+export varlist_2d=${varlist_2d:-"${PARMgfs}/product/gefs_shortparmlist_2d.parm"} #Parameter table for surface variables
+export varlist_3d=${varlist_3d:-"${PARMgfs}/product/gefs_shortparmlist_3d_h.parm"} #Parameter table for upper air instantaneous variables
+export varlist_3d_d=${varlist_3d_d:-"${PARMgfs}/product/gefs_shortparmlist_3d_d.parm"} #Parameter table for upper air daily-averaged variables
+export varlist_wav=${varlist_wav:-"${PARMgfs}/product/gefs_wav_shortparmlist.parm"} #Parameter table for wave variables
+export varlist_ocn_netcdf=${varlist_ocn_netcdf:-"${PARMgfs}/product/gefs_ocn_shortparmlist.parm"} #Parameter table for ocean netcdf variables
+export varlist_ice_netcdf=${varlist_ice_netcdf:-"${PARMgfs}/product/gefs_ice_shortparmlist.parm"} #Parameter table for ice netcdf variables
+export varlist_ocn_grib2=${varlist_ocn_grib2:-"${PARMgfs}/product/gefs_ocn_shortparmlist_grib2.parm"} #Parameter table ocn grib2 variables
+export varlist_ice_grib2=${varlist_ice_grib2:-"${PARMgfs}/product/gefs_ice_shortparmlist_grib2.parm"} #Parameter table ice grib2 variables
```

Comment:
There would be an issue if we were to condense these lists into two lists because one input data file (e.g. pgrb2b file) will have variables needed for both the f2d extracted variables and f3d extracted variables. If one input data file (e.g. pgrb2b file) has variables needed for just the f2d extracted variables, condensing the lists would not be an issue, but this is not the case.